src/Utils/systems.jl

```diff
@@ -15,6 +15,7 @@ import LazySets.constrained_dimensions
 
 *(M::AbstractMatrix, input::ConstantInput) =  ConstantInput(M * input.U)
 
+# TODO: kept for backwards-compatibility. To be removed.
 # no input: x' = Ax, x(0) = X0
 ContinuousSystem(A::AbstractMatrix, X0::LazySet) = IVP(LCS(A), X0)
```

Comment:
Do we have an issue for that already?

Reply:
Ok, in this issue i can just leave this part commented (or make another issue, but it's simpler to find there) until we have updated all benchmarks.